lib/chef/provider/mount.rb

```diff
@@ -24,7 +24,8 @@
 class Chef
   class Provider
     class Mount < Chef::Provider
-
+      Mount.provides :mount
```

Comment:
Copypasta.  It should never have to do Mount.provides unless you're doing it from outside the class.  I'll search through and fix.